docs/Project.toml

```diff
@@ -3,7 +3,7 @@ Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 ExpressionTreeForge = "93090adf-0e31-445f-8c8f-44d91f61d7ad"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
-ModelingToolkit = "961ee093-0014-501f-94e3-6117800e7a78"
+Symbolics = "0c5d862f-8b57-4792-8d23-62f2024744c7"
```

Comment:
You need to specify versions of packages that are not in stdlib.

Reply:
Is this not done in the Project.toml at the root in the [compat] field?

Reply:
Also in the docs if you use that module explicitly. If you only use it through ETF, there's no need to specify it here as a dependency.

Reply:
In the tutorial, I used `JuMP, MathOptInterface` and `Symbolics` in examples defining models.
The alternative of using `JuMP, MathOptInterface` and `Symbolics` is to replace them by `ADNLPModels, OptimizationProblems` and `JuMPNLPModels` and apply `get_expression_tree` from those models..